kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -178,12 +178,17 @@ private Object convertObject(Object kafkaConnectObject, Schema kafkaConnectSchem
     List<Field> kafkaConnectSchemaFields = kafkaConnectSchema.fields();
     Struct kafkaConnectStruct = (Struct) kafkaConnectObject;
     for (Field kafkaConnectField : kafkaConnectSchemaFields) {
-      Object bigQueryObject = convertObject(
-          kafkaConnectStruct.get(kafkaConnectField.name()),
-          kafkaConnectField.schema()
-      );
-      if (bigQueryObject != null) {
-        bigQueryRecord.put(kafkaConnectField.name(), bigQueryObject);
+      // ignore empty structures
+      boolean isEmptyStruct = kafkaConnectField.schema().type() == Schema.Type.STRUCT &&
```

Comment:
I'm a little confused about what's going on here or how it relates to the PR.

Reply:
Sometimes particular record types have fields with instances of classes with no fields (null objects, markers etc.).
Record and schema converters should omit such nested empty structs, otherwise BigQuery will raise an error.